app/stories/components/Dropdown.stories.tsx

```diff
@@ -5,6 +5,14 @@ import DropdownComponent from '@components/Dropdown'
 export default {
   title: 'Components',
   component: DropdownComponent,
+  argTypes: {
+    id: {
+      control: false,
+    },
+    handleChange: {
+      control: false,
+    },
+  },
```

Comment:
Notes for reviewers: Hide some controls for args that shouldn't be manually manipulated.